app/views/records/_form.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="card mb-3">
+  <%= form_with(model: record, local: true) do |form| %>
+    <div class="card-body">
+      <div class="row">
+        <div class="col-12">
+          <% if @record.errors.any? %>
+            <div id="error_explanation">
+              <h2>
+                <%= pluralize(@record.errors.count, "error") %> prohibited
+                this record from being saved:
```

Comment:
can we usar t('.error_massage') or only "impediram se ser salvo esse registro"

Reply:
Same comment about i18n usage. I'm keeping the way it is for now. Let's change this later if we agree that i18n should be addressed on the course. WDYT?

Reply:
Agree >_<